pvlib/test/test_spa.py

```diff
@@ -79,13 +79,15 @@
 Phi = 194.340241
 year = 1985 
 month = 2
-year_array = np.array([-499, 500, 1000, 1500, 1800, 1900, 1950, 1970, 1985, 1990, 2000, 2005])
-month_array = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12])
+year_array = np.array([-499, 500, 1000, 1500, 1800, 1860, 1900, 1950,
+                       1970, 1985, 1990, 2000, 2005])
+month_array = np.array([1, 2, 3, 4, 5, 6, 6, 7, 8, 9, 10, 11, 12])
```

Comment:
two `6`s?

Reply:
why not?
```python
np.arange(1,13)
# array([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12])
```

Reply:
Given how the tests are currently designed, `year_array` and `month_array` need to have the same number of elements. The test covering the years is doing pairs with `month_array`.
Adding one year to test lead to an additional month, that's why we have two 6 (could be something else). If we add 2050 and 2150, we will need two additional months. Or, we need to revamp the tests.

Reply:
Ah, thanks. I suggest we add the extra months to the array and a comment explaining the duplicates.